src/modules/party/Category.ts

```diff
@@ -11,6 +10,7 @@ import PokedexFilters from '../settings/PokedexFilters';
 import Settings from '../settings/Settings';
 
 export type PokemonCategory = {
+    id: KnockoutObservable<number>,
```

Comment:
Why observable? It shouldn't ever change, right?

Reply:
Probably not useful, indeed.

Reply:
Made it number.